cfsctl/handle.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"log"
+
+	pb "github.com/c-fs/cfs/proto"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+type handlerFunc func(context.Context, pb.CfsClient) error
+
+func rawHandle(ctx context.Context, fn handlerFunc) {
```

Comment:
let us make this as simple as possible for now. I actually would expect a simple setup function call that returns the gRPC client in each handler rather than a raw handler that intercepts each actual handling logic.